networkit/cpp/centrality/test/DynBetweennessGTest.cpp

```diff
@@ -69,6 +68,49 @@ TEST_F(DynBetweennessGTest, runDynApproxBetweennessSmallGraph) {
 
 }
 
+TEST_F(DynBetweennessGTest, runDynApproxBetweennessSmallGraph_edgeDeletion) {
+/* Graph:
+   0    3   6
+    \  / \ /
+     2    5
+    /  \ / \
+   1    4   7
+*/
+    int n = 8;
+    Graph G(n);
+
+    G.addEdge(0, 2);
+    G.addEdge(1, 2);
+    G.addEdge(2, 3);
+    G.addEdge(2, 4);
+    G.addEdge(3, 5);
+    G.addEdge(4, 5);
+    G.addEdge(5, 6);
+    G.addEdge(5, 7);
+
+    double epsilon = 0.1; // error
+    double delta = 0.1; // confidence
+    DynApproxBetweenness dynbc(G, epsilon, delta);
+    Betweenness bc(G);
+    dynbc.run();
+    bc.run();
+    std::vector<double> dynbc_scores = dynbc.scores();
+    std::vector<double> bc_scores = bc.scores();
+    for(int i=0; i<n; i++) {
+        DEBUG("Difference ", dynbc_scores[i]-bc_scores[i]/double(n*(n-1)));
```

Comment:
Instead of printing the difference, we should check that the results are the same.